frontend/components/product-list/ProductListBreadcrumb.tsx

```diff
@@ -40,10 +40,7 @@ export function ProductListBreadcrumb({
          separator={
             <Icon
                as={HiChevronRight}
-               display={{
-                  base: 'none',
-                  md: 'initial',
-               }}
+               display="initial"
```

Comment:
Do we need this anymore if we're never setting it to none?

Reply:
Nope good catch

Reply:
https://github.com/iFixit/react-commerce/pull/466/commits/ebb2f7d82e232fad6c85f2e63aec017ce62fa07a